### PR TITLE
add templates documentation, and fix one slight inaccuracy in anko mo…

### PR DESCRIPTION
**Pull request by floren**
…dule newEntry function doc

### CODE REVIEW

File: scripting/anko.md [hunk 1]
```diff
@@ -203,7 +203,7 @@ The following functions are only available in scripts implementing the `Main` fu
 * `readEntry() entry, error` returns the next entry and an error (if any). It will return an error when no entries remain.
 * `writeEntry(ent) error` writes out the given entry down the pipeline, returning an error if any.
 * `cloneEntry(ent) entry` returns a copy of the specified entry.
-* `newEntry() entry` creates an entirely new entry, with the timestamp set to the current time.
+* `newEntry() entry` creates an entirely new entry, with the timestamp set to the end of the current query.
```
Comment (traetox) [inline scripting/anko.md:206]: not true, the prototype for the function is: `func(ts time.Time, data []byte)`

**Review by traetox — CHANGES_REQUESTED** [review]
documentation of newEntry helper function isn't right

File: scripting/anko.md (hunk 1 above)
Comment (floren) [reply scripting/anko.md:206]: different functions in anko module vs. scheduled searches, see #96

**Comment by floren** [comment]
ptal

**Pull request merged**